Samples/word-blazor-add-in/word-blazor-add-in/Pages/HelloWorld.razor.js

```diff
@@ -0,0 +1,17 @@
+﻿/* Copyright(c) Maarten van Stam. All rights reserved. Licensed under the MIT License. */
+/**
+ * Basic function to show how to insert a Paragraph at the start of the Word Document
+ */
+export function insertParagraph() {
+
+    console.log("Hello JavaScript in Blazor!?!?!?");
+
+    return Word.run((context) => {
+
+        // insert a paragraph at the start of the document.
```

Comment:
```suggestion
        // Inserts a paragraph at the start of the document.
```
We should be consistent with tenses in comments. Also, add a period if comment is a complete sentence. I'm ok with leaving them off sentence fragments.